webview/src/webview_ios.mm

```diff
@@ -36,6 +36,7 @@ @interface WebViewDelegate : UIViewController <WKNavigationDelegate>
     @public int m_WebViewID;
     @public int m_RequestID;
     @public NSString *m_PendingUrl;
+    @public bool m_AllowSiteAction; // Added by dotGears/TrungB
```

Comment:
The documentation doesn't really add extra value.
Author can easily be found later (if needed).